includes/Modules/Analytics.php

```diff
@@ -662,6 +663,54 @@ protected function create_data_request( Data_Request $data ) {
 					);
 					return true;
 				};
+			case 'POST:create-account-ticket':
+				if ( ! isset( $data['accountName'] ) ) {
+					/* translators: %s: Missing parameter name */
+					return new WP_Error( 'missing_required_param', sprintf( __( 'Request parameter is empty: %s.', 'google-site-kit' ), 'accountName' ), array( 'status' => 400 ) );
+				}
+				if ( ! isset( $data['propertyName'] ) ) {
+					/* translators: %s: Missing parameter name */
+					return new WP_Error( 'missing_required_param', sprintf( __( 'Request parameter is empty: %s.', 'google-site-kit' ), 'propertyName' ), array( 'status' => 400 ) );
+				}
+				if ( ! isset( $data['profileName'] ) ) {
+					/* translators: %s: Missing parameter name */
+					return new WP_Error( 'missing_required_param', sprintf( __( 'Request parameter is empty: %s.', 'google-site-kit' ), 'profileName' ), array( 'status' => 400 ) );
+				}
+				if ( ! isset( $data['timezone'] ) ) {
+					/* translators: %s: Missing parameter name */
+					return new WP_Error( 'missing_required_param', sprintf( __( 'Request parameter is empty: %s.', 'google-site-kit' ), 'timezone' ), array( 'status' => 400 ) );
+				}
+
+				if ( ! $this->authentication->get_oauth_client()->using_proxy() ) {
+					return new WP_Error( 'requires_proxy', __( 'Analytics provisioning requires connecting via the Site Kit proxy.', 'google-site-kit' ), array( 'status' => 400 ) );
```

Comment:
We don't use the "proxy" term anywhere in user-facing language, let's refer to "Analytics provisioning requires connecting via Site Kit Service."

Reply:
Good point! Updated in https://github.com/google/site-kit-wp/pull/1512/commits/721ee981a9a5077e656c047e0923675527a468f2.